test/js/visual.js

```diff
@@ -132,4 +132,182 @@ describe("Image", function() {
             assert.strictEqual(image.callbacks, null);
         });
     });
+
+    describe("#_initialsBuilder", function() {

```

Comment:
@BeeMargarida I've moved these auxiliary methods under `ripe` (instead of `image`) given that they are logical methods and not really about displaying stuff on screen.

Please move the tests as well